resources/views/hardware/view.blade.php

```diff
@@ -678,9 +678,31 @@
                                             </div>
                                             <div class="col-md-6">
                                                 @can('superuser')
+                                             
                                                     <a href="{{ route('locations.show', ['location' => $asset->location->id]) }}">
                                                         {{ $asset->location->name }}
                                                     </a>
+
+                                                   @php  $asset_current_location = $asset->location;
+                                                            $depth = 0;
+                                                            $max_depth = 10;
+                                                   @endphp
+
+                                                    @while (($asset_current_location->parent_id != null))
+                                                        @if($depth >= $max_depth ) <<a href="{{ route('locations.show', ['location' => $asset_current_location->parent_id]) }}">
+                                                        ...
+                                                        </a>
+                                                        @else
+                                                    <<a href="{{ route('locations.show', ['location' => $asset_current_location->parent_id]) }}">
```

Comment:
Looks like a typo here? `<<`

Reply:
That's the path delineator.  I'm not sure if it needs additional html tags at that position since it inherits from <div class="col-md-6">

Reply:
We'd need to use the HTML entity for that then. `&lt;`.

Reply:
Ellipsis don't have any special meaning in HTML, whereas an open tag `<` does. I'd generally suggest using the correct HTML entities wherever possible, but ellipsis won't actually break HTML validation or rendering.